### DISCUSSION TITLE
A Tufte Inspired Manuscript with Typst

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#10472] [Show and Tell]
### Description

I hope you enjoy 😄 

https://github.com/fredguth/tufte-inspired

![tufte-inpired](https://github.com/user-attachments/assets/43c16ff2-43e5-4ac6-aee5-8d8c1b1e991c)

**Comment by mcanouil** [comment]
Nice work!

Feel free to submit to the following listing community-based listing of extensions:

- https://github.com/quarto-dev/quarto-web/tree/main/docs/extensions/listings
- https://github.com/mcanouil/quarto-extensions

**Reply by fredguth** [reply]
In https://github.com/quarto-dev/quarto-web/tree/main/docs/extensions/listings should I list as journal_article or custom_format?
It is a manuscript with custom typst and html formats.

**Reply by mcanouil** [reply]
That's a tricky question 😅
It's also why I drifted away from that schema in my listing website.

Note that it does not really matter as there are no quality control or whatever, so fill in whatever you think is best.
